### ISSUE TITLE
create symlink at /usr/local/bin/generate-schema on MacOS

### ISSUE THREAD

**Issue by bxparks** [bxparks/bigquery-schema-generator#8]
To install from PyPI, we use the following pip3 command:
$ sudo -H pip3 install bigquery-schema-generator

On Ubuntu (verified on 17.10, 16.04), the 'generate-schema' shell script is installed at: /usr/local/bin/generate-schema

On MacOS (verified on 10.13.2, using Python 3.6.4), the 'generate-schema' script is installed at:
/Library/Frameworks/Python.framework/Versions/3.6/bin/generate-schema
This is not an obvious location for the user.

We need to create a symlink from /usr/local/bin/generate-schema -> (the above location) on MacOS.

**Comment by bxparks** [comment]
After some Googling around, I found these:

- https://github.com/pypa/pip/issues/113
- https://trac.macports.org/ticket/50058

which basically says that behavior is expected (or even desired) in MacOS so that binaries from different versions don't clobber each other.

My Python 3.6.4 came from https://www.python.org/downloads/, which installs its pip3 in /usr/local/bin/pip3, which is a symlink to /Library/Frameworks/Python.framework/Versions/3.6/bin/pip3. Using this version of pip3, my `generate-schema` binary gets installed under the /Library/Frameworks/Python.framework/Versions/3.6/bin/ directory.

It seems that if I installed Python using MacPorts, those binaries get installed under /opt/local/Library/Frameworks/Python.framework/Versions/3.6/bin/, and using the MacPorts version of pip3 would install `generate-schema` under that directory.

I think the best I can do right now is add some of this information into the README.md, so that Python users on the Mac can figure out where to find the `generate-schema` shell script.

**Comment by bxparks** [comment]
Apparently the Python 3.6.4 installer modifies $HOME/.bash_profile to prepend /Library/Frameworks/Python.framework/Versions/3.6/bin/ into the $PATH variable. I didn't notice this because I had to logout and log back in for it to take effect. This means that the `generate-schema` script can be executed without its full path on MacOS as well.

I added much of this information in into the README.md file in v0.1.5 currently on PyPI.

Marking this closed.
